src/code/graph.c

```diff
@@ -32,6 +32,13 @@ UCodeInfo D_8012D248[3] = {
     { UCODE_S2DEX, gspS2DEX2d_fifoTextStart },
 };
 
+#ifdef ENABLE_MOTION_BLUR
+
+u16 (*gWorkBuf)[SCREEN_WIDTH * SCREEN_HEIGHT]; // pointer-to-array, array itself is allocated (see below)
+#define ALIGN64(val) (((val) + 0x3F) & ~0x3F)
```

Comment:
This is already defined here right?
https://github.com/HackerN64/HackerOoT/blob/bc3662f3af48fa1fafef2eeb2baa29d0d8936d2b/include/alignment.h#L7